src/Core/ConstraintValidator/CleanHtmlValidator.php

```diff
@@ -37,6 +37,18 @@
  */
 final class CleanHtmlValidator extends ConstraintValidator
 {
+    private const IFRAME_PATTERN = '/<[\s]*(i?frame|form|input|embed|object)/ims';
```

Comment:
It's not only iframe pattern :thinking:

Reply:
Well this is what PS currently checks for when setting allow Iframe is turned off. 

But yeah I can see why name can be confusing. But I am not sure about best name.
Perhaps on of those?
ADVANCED_HTML_PATTERN
THIRD_PARTY_HTML_PATTERN
EMBEDDABLE_HTML_PATTERN
RISKY_HTML_PATTERN
What do you think?